src/components/social-count/SocialCountSection.jsx

```diff
@@ -0,0 +1,22 @@
+import React, {useEffect, useState} from 'react';
+import {getWishlistSocialCount} from '../../swym/store-apis';
+import './SocialCountSection.css';
+
+/**
+ * @author swym
+ * @param {*} productData - This contains information about the product
```

Comment:
@gouravswym is this param definition correct? I see `socialCount` as param and not `productData`